demo/angularjs/components/search/doc.md

```diff
@@ -0,0 +1,17 @@
+# Search
+
+**Search allow users to specify a word or a phrase to find particular relevant pieces of content.**
+
+## Search
+
+<demo-block component="search" partial="search"></demo-block>
+
+## Suggestion
+
+<demo-block component="search" partial="suggestion"></demo-block>
+
+## Growing search
+
+Use growing search when space is constraint or when you don't want a prominent search.
```

Comment:
- `...space is *a* constraint`
- `don't`use contractions ಠ_ಠ
- `and search does not have to be proeminent` to avoid the `you`